### PR TITLE
Portfolio

### PR DESCRIPTION
**Pull request by marcusxavier85**
I THINK this is completely done.

### CODE REVIEW

File: portfolio.html [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+  
+</head>
+<body>
+
+	<nav class="navigationBar fontStyleHead">
+			<a href="#topJump" class="fontStyleHead">Home</a>
+			<a href="#aboutMeJump" class="fontStyleHead">About</a>
+			<a href="#portfolioJump" class="fontStyleHead">Portfolio</a>
+			<a href="#footerJump" class="fontStyleHead">Socials</a>
+		</div>
```
Comment (mporam) [inline portfolio.html]: these closing div tags are closing nothing? please remove

File: portfolio.html [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+		</div>
+		
+	</nav>
+	
+	<section id="topJump" class="mainImage">
+		<div class="backgroundImage">
```
Comment (mporam) [inline portfolio.html:21]: fix your indentation, child elements should be indented inside parents

File: portfolio.html [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+		</div>
+	</div>
+	</div>
+	</div>
+	</section>
+	
+	<section id="aboutMeJump"class="aboutMe">
+		<div class="aboutMeBox fontStyleHead">
+		<h2 class="aboutMeTitle">About Me</h2>
+		<img src="https://ca.slack-edge.com/T3CSKGCFN-U019S5TT2A1-1182c80c547c-512" alt="#" class="pictureOfOwner">
```
Comment (mporam) [inline portfolio.html]: why is the alt tag a #?

File: portfolio.html [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+				<a href="#"><img src="https://images.unsplash.com/photo-1599582723610-cd7a469c5016?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=500&q=60" alt="Yo" class="portfolioImage"></a>
+				<div class="leftText fontStyleHead">
+				<a href="#" class="portfolioText">Project is on the way!</a>
+				</div>
+			</div>
+			</div>
+			<div class="portfolioBox">
+				<div class="background">
+				<a href="#"><img src="https://images.pexels.com/photos/4887433/pexels-photo-4887433.jpeg?auto=compress&cs=tinysrgb&dpr=1&w=500" alt="Yo" class="portfolioImage"></a>
+				<div class="leftText fontStyleHead">
```
Comment (mporam) [inline portfolio.html]: left Text is not a semantic class name

File: portstyle.css [hunk 5]
```diff
@@ -0,0 +1,229 @@
...
+}
+
+.navigationBar a {
+	margin: 20px auto;
+	font-size: 1.5rem;
+	background: none;
+	border: none;
+	text-decoration: none;
+}
+
```
Comment (mporam) [inline portstyle.css:40]: extra line break

File: portstyle.css [hunk 6]
```diff
@@ -0,0 +1,229 @@
...
+.mainImage {
+	display: flex;
+	justify-content: center;
+	width: 100%;
+}
+
+
+.backgroundImage {
+	background-image: url("https://images.unsplash.com/photo-1592750088063-96b411cd5895?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1950&q=80");
```
Comment (mporam) [inline portstyle.css]: again, download this image, dont link to the online version

File: portstyle.css [hunk 7]
```diff
@@ -0,0 +1,229 @@
...
+	z-index: -20;
+	display: flex;
+	justify-content: center;
+	align-items: center;
+}
+
+.overlayText {
+	position: fixed;
+	border: 1px solid white;
+	padding: 0px 1%;
```
Comment (mporam) [inline portstyle.css]: dont need to put px on 0 values, 0 is 0 in any unit

File: portstyle.css [hunk 8]
```diff
@@ -0,0 +1,229 @@
...
+	border: 1px solid #c67e59;
+	height: 400px;
+}
+
+.background img {
+	object-fit: cover;
+	height: 200px;
+	width: 100%;
+}
+
```
Comment (mporam) [inline portstyle.css:138]: extra line break

File: portstyle.css [hunk 9]
```diff
@@ -0,0 +1,229 @@
...
+
+.portfolioText {
+	display: flex;
+	justify-content: center;
+	text-decoration: none;
+	color: #FFFFFF;
+
```
Comment (mporam) [inline portstyle.css]: line break?

File: portstyle.css [hunk 10]
```diff
@@ -0,0 +1,229 @@
...
+	justify-content: center;
+	text-decoration: none;
+	color: #FFFFFF;
+	
+}
+
+.portfolioText a {
+	text-decoration: none;
+}
+
```
Comment (mporam) [inline portstyle.css:148]: line break?

File: portstyle.css [hunk 11]
```diff
@@ -0,0 +1,229 @@
...
+@media screen and (max-width: 1100px) {
+	
+	.threeBoxLayout {
+		display: flex;
+		width: 100%;
+		grid-template-columns: 100%;
+		flex-direction: column;
+
```
Comment (mporam) [inline portstyle.css]: line break?

File: portstyle.css [hunk 12]
```diff
@@ -0,0 +1,229 @@
...
+		width: 100%;
+		grid-template-columns: 100%;
+		flex-direction: column;
+		
+	}
+}
+	
+@media screen and (max-width: 1100px) {
+	.aboutMe {
+			display: grid;
```
Comment (mporam) [inline portstyle.css]: fix indentation

File: portstyle.css [hunk 13]
```diff
@@ -0,0 +1,229 @@
...
+	
+
+@media screen and (max-width: 400px) {
+	.navigationBar a {
+		padding: 3px;
+		margin: 1%;
+		
+	}
+	.overlayText {
+		top: 41%;
```
Comment (mporam) [inline portstyle.css]: why? what is the purpose of this?

File: portstyle.css [hunk 14]
```diff
@@ -0,0 +1,229 @@
...
+			grid-template-columns: 1fr;
+		}
+}
+	
+
+@media screen and (max-width: 400px) {
+	.navigationBar a {
+		padding: 3px;
+		margin: 1%;
+
```
Comment (mporam) [inline portstyle.css]: line break?

**Review by mporam — CHANGES_REQUESTED** [review]
the social media image in the footer are distorted and the twitter one doesnt work.

File: portfolio.html (hunk 1 above)
Comment (marcusxavier85) [reply portfolio.html]: Removed.

File: portfolio.html (hunk 2 above)
Comment (marcusxavier85) [reply portfolio.html:21]: Done

File: portfolio.html (hunk 3 above)
Comment (marcusxavier85) [reply portfolio.html]: Was just a placeholder I forgot to change. Changed to Picture of Marcus.

File: portfolio.html (hunk 4 above)
Comment (marcusxavier85) [reply portfolio.html]: Been changed to sixBoxText.

File: portstyle.css (hunk 5 above)
Comment (marcusxavier85) [reply portstyle.css:40]: Removed.

File: portstyle.css (hunk 7 above)
Comment (marcusxavier85) [reply portstyle.css]: Removed.

File: portstyle.css (hunk 8 above)
Comment (marcusxavier85) [reply portstyle.css:138]: Removed.

File: portstyle.css (hunk 9 above)
Comment (marcusxavier85) [reply portstyle.css]: Removed.

File: portstyle.css (hunk 10 above)
Comment (marcusxavier85) [reply portstyle.css:148]: Removed.

File: portstyle.css (hunk 11 above)
Comment (marcusxavier85) [reply portstyle.css]: REmoved.

File: portstyle.css (hunk 12 above)
Comment (marcusxavier85) [reply portstyle.css]: Fixed.

File: portstyle.css (hunk 13 above)
Comment (marcusxavier85) [reply portstyle.css]: It was originally to keep the .overlayText in place for when it was shrunk down to mobile. I have just tried removing it and it works anyway so I ahve removed.

File: portstyle.css (hunk 6 above)
Comment (marcusxavier85) [reply portstyle.css]: Done for all images.

File: portfolio.html [hunk 15]
```diff
@@ -10,103 +10,105 @@
...
 	
 	<section id="topJump" class="mainImage">
 		<div class="backgroundImage">
-		<div class="overlayText">
-			<h1 class="fontStyleHead">Marcus Bowyer</h1>
-		<div class="secondOverlayText">
-			<h2 class="fontStyleHead">Full Stack Web Devloper</h2>
+			<div class="overlayText">
+				<h1 class="fontStyleHead">Marcus Bowyer</h1>
+			<div class="secondOverlayText">
```
Comment (mporam) [inline portfolio.html]: indentation here is still not correct

File: portfolio.html [hunk 16]
```diff
@@ -10,103 +10,105 @@
...
-	</div>
 	</section>
 	
 	<section id="aboutMeJump"class="aboutMe">
 		<div class="aboutMeBox fontStyleHead">
-		<h2 class="aboutMeTitle">About Me</h2>
-		<img src="https://ca.slack-edge.com/T3CSKGCFN-U019S5TT2A1-1182c80c547c-512" alt="#" class="pictureOfOwner">
-	</div>
+			<h2 class="aboutMeTitle">About Me</h2>
+			<img src="file:///Users/academy/Documents/T3CSKGCFN-U019S5TT2A1-1182c80c547c-512.jpg" alt="Picture of Marcus" class="pictureOfOwner">
```
Comment (mporam) [inline portfolio.html]: save the image in the same folder as your html file, then just put the image name here

File: portstyle.css [hunk 17]
```diff
@@ -28,23 +32,20 @@ body {
...
 .mainImage {
 	display: flex;
 	justify-content: center;
 	width: 100%;
 }
 
-
 .backgroundImage {
-	background-image: url("https://images.unsplash.com/photo-1592750088063-96b411cd5895?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1950&q=80");
+	background-image: url("file:///Users/academy/Documents/photo-1592750088063-96b411cd5895.htm");
```
Comment (mporam) [inline portstyle.css]: this needs to be a relative url, put the image in the same folder as your html and then just link the image file name

**Review by mporam — CHANGES_REQUESTED** [review]

File: portfolio.html (hunk 16 above)
Comment (marcusxavier85) [reply portfolio.html]: Done for both.

File: portstyle.css (hunk 17 above)
Comment (marcusxavier85) [reply portstyle.css]: Done for both.

File: portfolio.html (hunk 15 above)
Comment (marcusxavier85) [reply portfolio.html]: I believe everything is indeneted correctly. I also moved back my sixBoxLayout section as it was out of line with all the other HTML.

**Review by mporam — APPROVED** [review]

**Pull request merged**